### PR TITLE
use new sessionid format

### PR DESCRIPTION
**Pull request by goenning**
Hey @NikTheNak if you could have a look at this PR please.

This PR changes to a new session id format that will enable some neat server-side optimisations in near future.

I don't have unity installed, so I couldn't test and I don't know if the `.meta` files have to be updated too.

Thanks!

### CODE REVIEW

File: Runtime/Aptabase.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public static class Aptabase
         private static string _baseURL;
         
         private static readonly TimeSpan _sessionTimeout = TimeSpan.FromMinutes(60);
+        private static readonly Random _random = new();
```
Comment (NickDLion) [inline Runtime/Aptabase.cs]: We can do 
using Random = UnityEngine.Random instead of defining this

File: Runtime/Aptabase.cs [hunk 2]
```diff
@@ -165,6 +166,11 @@ private static int GetFlushInterval()
             return _env.isDebug ? 2000 : 60000;
         }
         
-        private static string NewSessionId() => Guid.NewGuid().ToString().ToLower();
+        public static string NewSessionId()
+        {
+            var epochInSeconds = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+            var random = _random.NextInt64(0, 99999999);
+            return (epochInSeconds * 100000000 + random).ToString();
```
Comment (NickDLion) [inline Runtime/Aptabase.cs:173]: We can also use
var epochInSeconds = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
            return (epochInSeconds * 100000000 + Random.Range(0,99999999)).ToString();

**Comment by goenning** [comment]
Thanks @NickDeLaRosaLion want to have another look?
We don't need to touch the `.meta` files?

File: Runtime/Aptabase.cs (hunk 2 above)
Comment (NikTheNak) [reply Runtime/Aptabase.cs:173]: Oops wrong account lol
new changes should be okay, but you need
`using Random = UnityEngine.Random;` defined at the top
can also change public to private as we aren't getting the new id anywhere else
.meta file is fine

Comment (goenning) [reply Runtime/Aptabase.cs:173]: Oh, I assumed `Random` was a static class in the `UnityEngine` namespace, but I suppose `UnityEngine.Random` is a namespace too?

Updated :)

**Pull request merged**